src/sentry/runner/commands/devserver.py

```diff
@@ -18,10 +18,11 @@
 @click.option('--watchers/--no-watchers', default=True, help='Watch static files and recompile on changes.')
 @click.option('--workers/--no-workers', default=False, help='Run asynchronous workers.')
 @click.option('--browser-reload/--no-browser-reload', default=False, help='Automatic browser refreshing on webpack builds')
+@click.option('--environment', default='development', help='The environment name.')
```

Comment:
Sorry, and if you did care to do this, you can add `type=click.Choice(('development', 'production', ...))`